ahcore/callbacks/h5_callback.py

```diff
@@ -10,15 +10,15 @@
 from dlup.data.dataset import ConcatDataset, TiledWsiDataset
 from pytorch_lightning import Callback
 
-from ahcore.callbacks.utils import _get_h5_output_filename
+from ahcore.callbacks.utils import InferencePrecision, _get_h5_output_filename, NormalizationType
 from ahcore.utils.data import DataDescription, GridDescription
 from ahcore.utils.io import get_logger
 from ahcore.utils.types import GenericArray
 from ahcore.writers import H5FileImageWriter
 
 
 class WriteH5Callback(Callback):
-    def __init__(self, max_queue_size: int, max_concurrent_writers: int, dump_dir: Path):
+    def __init__(self, max_queue_size: int, max_concurrent_writers: int, dump_dir: Path, normalization_type: str):
```

Comment:
In here there is now only an argument for the normalization and not for the precision. The precision is now set to uint8 without choice to the user. I think an extra parameter will be necessary, but then we add many arguments. (maybe also not so nice @jonasteuwen would you add more arguments here/another way to sovle this?)

Reply:
@jonasteuwen I like Marek's comment above. I think we should make the precision as a consumable parameter of the H5 callback's constructor.

Reply:
@moerlemans it's fixed now. Are there any other comments?